R/fit-dcm.R

```diff
@@ -72,6 +72,10 @@
 #'     [CmdStanModel](https://mc-stan.org/cmdstanr/reference/CmdStanModel.html)
 #'     class.
 #'
+#' @concept Bayesian
+#' @concept Stan
+#' @concept Q-matrix
```

Comment:
Proper nouns used in the middle of sentences are added as `@concept`s and added to the allow-list.